internal/core/graphics/image.rs

```diff
@@ -20,15 +20,16 @@ mod htmlimage;
 mod svg;
 
 #[allow(missing_docs)]
-#[vtable::vtable]
+#[cfg_attr(not(feature = "ffi"), i_slint_core_macros::remove_extern)]
+#[vtable::vtable(no_extern)]
```

Comment:
Since this is used unconditionally and all fn fields have an ABI, perhaps it would be a better API if it was called `require_abi` and error out if a `fn` has no ABI?

Reply:
Ahh yes, indeed, that would make it a little awkward. Ok, never mind - I'm just (unintentionally) nitpicking :)